### PR TITLE
Persist panel options buttons when menu is open

### PR DESCRIPTION
**Pull request by jwlee64**
https://github.com/wandb/weave/assets/25037002/25fa2738-d8a5-40f3-b6f0-9a6f9d4b2b41

https://wandb.atlassian.net/browse/WB-14141

### CODE REVIEW

File: weave-js/src/components/Panel2/ChildPanel.tsx [hunk 1]
```diff
@@ -630,6 +631,8 @@ export const ChildPanel: React.FC<ChildPanelProps> = props => {
                       <IconOverflowHorizontal />
                     </IconButton>
                   }
+                  onOpen={() => setIsMenuOpen(true)}
```
Comment (tssweeney) [inline weave-js/src/components/Panel2/ChildPanel.tsx:634]: This pattern edges on violating the single source of truth principle of controlled and uncontrolled components: https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components.

Typically if you are going to manage the open state of a popup, you want to pass it in as well. This Popup uses Semantic's popup. See https://react.semantic-ui.com/modules/popup/#usage-controlled for details on this. 

In short, this will work, but there is a possibility of getting out of sync. Consider allowing caller to make this controlled by passing the popup state into the popup directly.

**Review by tssweeney — APPROVED** [review]
Nice!
